python/cudf/cudf/core/reshape.py

```diff
@@ -1160,3 +1160,271 @@ def _length_check_params(obj, columns, name):
                 f"length of the columns being "
                 f"encoded ({len(columns)})."
             )
+
+
+def _get_names(arrs, names, prefix: str = "row"):
```

Comment:
Don't specify a default for `row`, this should always be called with an explicit prefix. Also rename this to `_get_pivot_names` for clarity in how it is used.
```suggestion
def _get_pivot_names(arrs, names, prefix):
```

Reply:
Both changes done.